### ISSUE TITLE
Go compiler as overlay bundle

### ISSUE THREAD

**Issue by ivandavidov** [ivandavidov/minimal#155]
Research if it is possible to add [Go](https://golang.org/) as overlay bundle.

* Are the Go tools statically linked? If not, how easy/hard is it to build Go from source code?
* Is is possible to compile/run simple "hello world" program in debootstrap/chroot environment?
* Additional host requirements for ``go get``, e.g. git, wget, etc.?
* Any dependencies on GLIBC link libraries (*.a)? If so, how easy/hard is it to add them in MLL?

**Comment by ivandavidov** [comment]
``Go`` works fine in MLL container environment. I tried it with ``glibc_full`` as dependency but it should work fine with just ``libpthread``. I was able to compile and run successfully the [one-time pad](https://github.com/ivandavidov/go-lang/blob/master/onetimepad.go) program which I created recently.

I'm optimist that ``Go`` will work fine as part of the MLL OS, once it is properly packaged as overlay bundle.

This could also be seen as resolution for the long standing issue #59.

**Note:** the command ``go get`` doesn't work because it depends on other tools (e.g. git, svn, etc.) which are not present in MLL.
